pom.xml

```diff
@@ -80,6 +80,9 @@
         <!-- Form Recognizer -->
         <module>sdk/formrecognizer/azure-ai-formrecognizer</module>
 
+        <!-- Personalizer -->
+        <module>sdk/personalizer/azure-ai-personalizer</module>
```

Comment:
The only entry you should need in this file is the module entry down past line 900. @srnagar is in the process of removing these other entries for Jacoco/Javadoc etc. as they were originally in there for rollup reports. These files are only supposed to be aggregate pom files. You shouldn't need to add anything to this file other than the module entry below.